CMS theme rebased 4

marcoscv-work opened the pull request.

marco-leo committed:
LPD-56159 Index CMS information about entries/folders

marco-leo committed:
LPD-56159 Use new properties to simplify the filter logic

marco-leo committed:
LPD-56159 Not needed, passed as request parameter now

marco-leo committed:
LPD-56159 Provide parent folder to the UI components

adolfopa committed:
LPD-56159 Move method to base class

adolfopa committed:
LPD-56159 Move to model document contributor

adolfopa committed:
LPD-56159 Add dropdowns only when operating inside a folder

adolfopa committed:
LPD-56159 Provide parent folder when creating folder

adolfopa committed:
LPD-56159 Provide parent folder when creating content

adolfopa committed:
LPD-56159 Avoid adding properties if not a CMS object

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

ambrinchaudhary committed:
LPD-56159 AdditionalProps is not needed, now it refresh when a folder is created

marcoscv-work commented:
ci:test:sf

liferay-continuous-integration commented:
Test suite: ci:test:sf has been triggered.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-10.liferay.com/job/test-portal-source-format/6063/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-10.liferay.com/userContent/jobs/test-portal-source-format/builds/6063/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 37 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/74506ae98709219bfca788f6a20ee6b18a7f7f07">74506ae98709219bfca788f6a20ee6b18a7f7f07</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/marcoscv-work/liferay-portal/tree/cms-theme-rebased-4">cms-theme-rebased-4</a><br/>Branch GIT ID: <a href="https://github.com/marcoscv-work/liferay-portal/commit/69aa80e6dc3e9199d273d9c4b3d23ebd9a4ea549">69aa80e6dc3e9199d273d9c4b3d23ebd9a4ea549</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-10.liferay.com/job/test-portal-source-format/6063/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-10.liferay.com/userContent/jobs/test-portal-source-format/builds/6063/jenkins-report.html">here</a>.</h5><pre>     [exec] &gt; Task :packageRunCheckFormat
     [exec] yarn run v1.22.22
     [exec] \$ node-scripts check:ci
     [exec] 
     [exec] ?????? Running preflight checks...
     [exec] 
     [exec]    ?? BAD - node-scripts.config.js is out of date, run `yarn run node-scripts generate:global-config`
     [exec] ?????? Checking outdated tsconfig.json files ...
     [exec] 
     [exec] ?????? Running TypeScript checks on modified files...
     [exec] ?????? A total of 8 CPUs were detected: launching tsc using 8 workers
     [exec] ??? Checked apps/site/site-cms-site-initializer
     [exec] 
     [exec] ?????? Running format checks on modified files...
     [exec]    /opt/dev/projects/github/liferay-portal/modules/apps/frontend-theme/frontend-theme-cms/src/css/_clay_variables.scss
     [exec]      1:1  error  File has format errors.  (format check)
     [exec]    
     [exec]    ??? 1 problem (1 error, 0 warnings)
     [exec]    
     [exec]    
     [exec] ??? CI checks failed.
     [exec] error Command failed with exit code 1.
     [exec] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
     [exec] 
     [exec] &gt; Task :packageRunCheckFormat FAILED
     [exec] 
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':packageRunCheckFormat'.
     [exec] &gt; Process 'command '/opt/dev/projects/github/liferay-portal/build/node/bin/node'' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] &gt; Run with --info or --debug option to get more log output.
     [exec] &gt; Run with --scan to get full insights.
     [exec] &gt; Get more help at https://help.gradle.org.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':packageRunCheckFormat'.
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda\$executeIfValid\$1(ExecuteActionsTaskExecuter.java:148)
     [exec]   at org.gradle.internal.Try\$Failure.ifSuccessfulOrElse(Try.java:282)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:146)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:134)
     [exec]   at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-10.liferay.com/job/test-portal-source-format/6063/">test-portal-source-format#6063</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-10.liferay.com/userContent/jobs/test-portal-source-format/builds/6063/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/marcoscv-work/liferay-portal/pull/370">marcoscv-work#370</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/223054037">[master] ci:test:sf - marcoscv-work &gt; marcoscv-work - PR#370 - 2025-05-22[04:11:23]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>223054037</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-10.liferay.com/job/publish-testray-report/6790/">publish-testray-report#6790</a><br/></div></div>

marcoscv-work commented:
ci:test:sf

liferay-continuous-integration commented:
Test suite: ci:test:sf has been triggered.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-40.liferay.com/job/test-portal-source-format/8395/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-source-format/builds/8395/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-source-format/8395/">test-portal-source-format#8395</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-source-format/builds/8395/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/marcoscv-work/liferay-portal/pull/370">marcoscv-work#370</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/224226790">[master] ci:test:sf - marcoscv-work &gt; marcoscv-work - PR#370 - 2025-05-23[02:19:38]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>224226790</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/publish-testray-report/31905/">publish-testray-report#31905</a><br/></div></div>

marcoscv-work commented:
ci:test:sf

liferay-continuous-integration commented:
Test suite: ci:test:sf has been triggered.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-27.liferay.com/job/test-portal-source-format/8239/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/8239/jenkins-report.html">here</a>.</b></p>

adolfopa committed:
LPD-56159 Integration tests

I'm removing the testGetAPIURL test, because it isn't testing
behaviour, but implementation, so it is breaking every single time
that we change anything in the api url (even if correct).

marcogalluzzi committed:
LPD-56159 Fix playwright test by adding the destination object entry folder

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 26 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/3b3b1680a0697e1f1282aac74b489d4edfe79d2c">3b3b1680a0697e1f1282aac74b489d4edfe79d2c</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/marcoscv-work/liferay-portal/tree/cms-theme-rebased-4">cms-theme-rebased-4</a><br/>Branch GIT ID: <a href="https://github.com/marcoscv-work/liferay-portal/commit/0a29a1da503323de85fc43b191e51427d6b0faac">0a29a1da503323de85fc43b191e51427d6b0faac</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-source-format/8239/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/8239/jenkins-report.html">here</a>.</h5><pre>format-source-files:
     [java] Loading file:/opt/dev/projects/github/liferay-portal/portal-impl/classes/system.properties
     [java] java.lang.Exception: Found 1 formatting issues:
     [java] 1: ./modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/BaseSectionDisplayContext.java expected:&lt;...bjectDefinitionId(),[ "&amp;plid=",
     [java]               themeDisplay.getPlid(), "&amp;redirect=",
     [java]               ]themeDisplay.getURLC...&gt; but was:&lt;...bjectDefinitionId(),[
     [java]               "&amp;plid=", themeDisplay.getPlid(),
     [java]               "&amp;redirect=", ]themeDisplay.getURLC...&gt;
     [java] 
     [java]   at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:474)
     [java]   at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:302)
[stopwatch] [run.batch.test.action: 24:47.475 sec]
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:449: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:651: Java returned: 1
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null812685338.properties

merge-test-results:
    [mkdir] Created dir: /opt/dev/projects/github/liferay-portal/test-results
[junitreport] Processing /opt/dev/projects/github/liferay-portal/test-results/TESTS-TestSuites.xml to /tmp/null245512479
[junitreport] Loading stylesheet ile:/opt/java/ant/lib/ant-junit.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
[junitreport] Transform time: 199ms
[junitreport] Deleting: /tmp/null245512479
     [echo] A report with all the test results can be found at test-results/html/index.html.
    [mkdir] Created dir: /opt/dev/projects/github/liferay-jenkins-ee/test-results
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-jenkins-ee/test-results
     [echo] run.batch.test.tear.down.start.timestamp: 05-23-2025 03:40:41:835 PDT
[stopwatch] [run.batch.test.tear.down: 0.001 sec]
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null812685338.properties

print-gc-logs:
     [echo] 
     [echo] ##
     [echo] ## /tmp/ant-gc.log
     [echo] ##
     [echo] 
[beanshell] Java HotSpot(TM) 64-Bit Server VM (25.381-b09) for linux-amd64 JRE (1.8.0_381-b09), built on Jun 14 2023 13:47:37 by "java_re" with gcc 7.3.0</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-27.liferay.com/job/test-portal-source-format/8239/">test-portal-source-format#8239</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/8239/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/marcoscv-work/liferay-portal/pull/370">marcoscv-work#370</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/224232722">[master] ci:test:sf - marcoscv-work &gt; marcoscv-work - PR#370 - 2025-05-23[03:14:24]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>224232722</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-27.liferay.com/job/publish-testray-report/31350/">publish-testray-report#31350</a><br/></div></div>

marcoscv-work commented:
ci:test:sf

liferay-continuous-integration commented:
Test suite: ci:test:sf has been triggered.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-37.liferay.com/job/test-portal-source-format/13020/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/13020/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 34 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/2fad120d2412b8589bc8557ba26d75b5a7ef3cf6">2fad120d2412b8589bc8557ba26d75b5a7ef3cf6</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/marcoscv-work/liferay-portal/tree/cms-theme-rebased-4">cms-theme-rebased-4</a><br/>Branch GIT ID: <a href="https://github.com/marcoscv-work/liferay-portal/commit/32b8dff61726099f2b1834633e511b7e7fea879c">32b8dff61726099f2b1834633e511b7e7fea879c</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-37.liferay.com/job/test-portal-source-format/13020/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/13020/jenkins-report.html">here</a>.</h5><pre>format-source-files:
     [java] Loading file:/opt/dev/projects/github/liferay-portal/portal-impl/classes/system.properties
     [java] java.lang.Exception: Found 1 formatting issues:
     [java] 1: ./modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/internal/upgrade/v3_2_11/test/DLFileEntryTypesDDMStructureUpgradeProcessTest.java expected:&lt;...eStepRegistrator;
     [java] 
     [java]   [private Group _companyGroup;
     [java] 
     [java]   @Inject
     [java]   private CompanyLocalService _companyLocalService;
     [java] 
     [java]   @Inject
     [java]   private DDMStructureLocalService _ddmStructureLocalService;
     [java] 
     [java]   @Inject
     [java]   private DLFileEntryTypeService _dlFileEntryTypeService;
     [java] 
     [java]   @Inject
     [java]   private EntityCache _entityCache];
     [java] 
     [java]   @DeleteAfterTest...&gt; but was:&lt;...eStepRegistrator;
     [java] 
     [java]   [@Inject
     [java]   private CompanyLocalService _companyLocalService;
     [java] 
     [java]   @Inject
     [java]   private DDMStructureLocalService _ddmStructureLocalService;
     [java] 
     [java]   @Inject
     [java]   private DLFileEntryTypeService _dlFileEntryTypeService;
     [java] 
     [java]   @Inject
     [java]   private EntityCache _entityCache;
     [java] 
     [java]   private Group _companyGroup];
     [java] 
     [java]   @DeleteAfterTest...&gt;
     [java] 
     [java]   at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:474)
     [java]   at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:302)
[stopwatch] [run.batch.test.action: 32:16.204 sec]
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:421: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:623: Java returned: 1
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null384582484.properties

merge-test-results:
    [mkdir] Created dir: /opt/dev/projects/github/liferay-portal/test-results
[junitreport] Processing /opt/dev/projects/github/liferay-portal/test-results/TESTS-TestSuites.xml to /tmp/null1726973899
[junitreport] Loading stylesheet ile:/opt/java/ant/lib/ant-junit.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
[junitreport] Transform time: 201ms</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-source-format/13020/">test-portal-source-format#13020</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/13020/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/marcoscv-work/liferay-portal/pull/370">marcoscv-work#370</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/224740680">[master] ci:test:sf - marcoscv-work &gt; marcoscv-work - PR#370 - 2025-05-23[14:34:42]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>224740680</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/publish-testray-report/48330/">publish-testray-report#48330</a><br/></div></div>

marcoscv-work commented:
ci:test:sf

liferay-continuous-integration commented:
Test suite: ci:test:sf has been triggered.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-9.liferay.com/job/test-portal-source-format/6798/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-9.liferay.com/userContent/jobs/test-portal-source-format/builds/6798/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 31 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/571b118126ef763b234f80b778fd55ca8defd69c">571b118126ef763b234f80b778fd55ca8defd69c</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/marcoscv-work/liferay-portal/tree/cms-theme-rebased-4">cms-theme-rebased-4</a><br/>Branch GIT ID: <a href="https://github.com/marcoscv-work/liferay-portal/commit/93cf33f80ca346afb0008f01117893b5c2c46ec8">93cf33f80ca346afb0008f01117893b5c2c46ec8</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-9.liferay.com/job/test-portal-source-format/6798/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-9.liferay.com/userContent/jobs/test-portal-source-format/builds/6798/jenkins-report.html">here</a>.</h5><pre>format-source-files:
     [java] Loading file:/opt/dev/projects/github/liferay-portal/portal-impl/classes/system.properties
     [java] java.lang.Exception: Found 5 formatting issues:
     [java] 1: ./bun dles/jboss-eap-8.0.0/.installation/manifest_version.yaml expected:&lt;-[ groupId: org.jboss.eap.channels
     [java]     artifactId: eap-8.0
     [java]     description:
     [java]         "Manifest for EAP 8.0"
     [java]     version: 1.0.1.GA-redhat-00003
     [java] maven:
     [java] schemaVersion: 1.0.0]&gt; but was:&lt;-[--
     [java] schemaVersion: "1.0.0"
     [java] maven:
     [java] - groupId: "org.jboss.eap.channels"
     [java]   artifactId: "eap-8.0"
     [java]   version: "1.0.1.GA-redhat-00003"
     [java]   description: "Manifest for EAP 8.0"
     [java] ]&gt;
     [java] 2: ./bun dles/jboss-eap-8.0.0/.installation/installer-channels.yaml expected:&lt;[manifest:
     [java]     maven:
     [java]         artifactId: eap-8.0
     [java]         groupId: org.jboss.eap.channels
     [java] name: eap-8.0
     [java] noStreamStrategy: none
     [java] repositories:
     [java]     -   id: mrrc
     [java]         url: https://maven.repository.redhat.com/ga/
     [java] schemaVersion: 2.0.0]&gt; but was:&lt;[---
     [java] schemaVersion: "2.0.0"
     [java] name: "eap-8.0"
     [java] repositories:
     [java]   - id: "mrrc"
     [java]     url: "https://maven.repository.redhat.com/ga/"
     [java] manifest:
     [java]   maven:
     [java]     groupId: "org.jboss.eap.channels"
     [java]     artifactId: "eap-8.0"
     [java] noStreamStrategy: "none"
     [java] ]&gt;
     [java] 3: Truncated message :
     [java] ./bun dles/jboss-eap-8.0.0/.installation/manifest.yaml expected:&lt;[schemaVersion: 1.0.0
     [java] streams:
     [java]     -   artifactId: aws-java-sdk-core
     [java]         groupId: com.amazonaws
     [java]         version: 1.12.284.redhat-00002
     [java]     -   artifactId: aws-java-sdk-kms
     [java]         groupId: com.amazonaws
     [java]         version: 1.12.284.redhat-00002
     [java]     -   artifactId: aws-java-sdk-s3
     [java]         groupId: com.amazonaws
     [java]         version: 1.12.284.redhat-00002
     [java]     -   artifactId: jmespath-java
     [java]         groupId: com.amazonaws
     [java]         version: 1.12.284.redhat-00002
     [java]     -   artifactId: hppc
     [java]         groupId: com.carrotsearch
     [java]         version: 0.8.1.redhat-00001
     [java]     -   artifactId: classmate</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-9.liferay.com/job/test-portal-source-format/6798/">test-portal-source-format#6798</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-9.liferay.com/userContent/jobs/test-portal-source-format/builds/6798/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/marcoscv-work/liferay-portal/pull/370">marcoscv-work#370</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/226033913">[master] ci:test:sf - marcoscv-work &gt; marcoscv-work - PR#370 - 2025-05-26[07:58:08]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>226033913</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-9.liferay.com/job/publish-testray-report/7577/">publish-testray-report#7577</a><br/></div></div>

marcoscv-work commented:
ci:test:sf

liferay-continuous-integration commented:
Test suite: ci:test:sf has been triggered.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-1.liferay.com/job/test-portal-source-format/4487/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-1.liferay.com/userContent/jobs/test-portal-source-format/builds/4487/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 34 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/3527940d19a039bfb2056c6f8ff9fa12549fa942">3527940d19a039bfb2056c6f8ff9fa12549fa942</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/marcoscv-work/liferay-portal/tree/cms-theme-rebased-4">cms-theme-rebased-4</a><br/>Branch GIT ID: <a href="https://github.com/marcoscv-work/liferay-portal/commit/93cf33f80ca346afb0008f01117893b5c2c46ec8">93cf33f80ca346afb0008f01117893b5c2c46ec8</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-1.liferay.com/job/test-portal-source-format/4487/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-1.liferay.com/userContent/jobs/test-portal-source-format/builds/4487/jenkins-report.html">here</a>.</h5><pre>format-source-files:
     [java] Loading file:/opt/dev/projects/github/liferay-portal/portal-impl/classes/system.properties
     [java] java.lang.Exception: Found 5 formatting issues:
     [java] 1: ./bun dles/jboss-eap-8.0.0/.installation/installer-channels.yaml expected:&lt;[manifest:
     [java]     maven:
     [java]         artifactId: eap-8.0
     [java]         groupId: org.jboss.eap.channels
     [java] name: eap-8.0
     [java] noStreamStrategy: none
     [java] repositories:
     [java]     -   id: mrrc
     [java]         url: https://maven.repository.redhat.com/ga/
     [java] schemaVersion: 2.0.0]&gt; but was:&lt;[---
     [java] schemaVersion: "2.0.0"
     [java] name: "eap-8.0"
     [java] repositories:
     [java]   - id: "mrrc"
     [java]     url: "https://maven.repository.redhat.com/ga/"
     [java] manifest:
     [java]   maven:
     [java]     groupId: "org.jboss.eap.channels"
     [java]     artifactId: "eap-8.0"
     [java] noStreamStrategy: "none"
     [java] ]&gt;
     [java] 2: ./bun dles/jboss-eap-8.0.0/.installation/manifest_version.yaml expected:&lt;-[ groupId: org.jboss.eap.channels
     [java]     artifactId: eap-8.0
     [java]     description:
     [java]         "Manifest for EAP 8.0"
     [java]     version: 1.0.1.GA-redhat-00003
     [java] maven:
     [java] schemaVersion: 1.0.0]&gt; but was:&lt;-[--
     [java] schemaVersion: "1.0.0"
     [java] maven:
     [java] - groupId: "org.jboss.eap.channels"
     [java]   artifactId: "eap-8.0"
     [java]   version: "1.0.1.GA-redhat-00003"
     [java]   description: "Manifest for EAP 8.0"
     [java] ]&gt;
     [java] 3: Truncated message :
     [java] ./bun dles/jboss-eap-8.0.0/.installation/manifest.yaml expected:&lt;[schemaVersion: 1.0.0
     [java] streams:
     [java]     -   artifactId: aws-java-sdk-core
     [java]         groupId: com.amazonaws
     [java]         version: 1.12.284.redhat-00002
     [java]     -   artifactId: aws-java-sdk-kms
     [java]         groupId: com.amazonaws
     [java]         version: 1.12.284.redhat-00002
     [java]     -   artifactId: aws-java-sdk-s3
     [java]         groupId: com.amazonaws
     [java]         version: 1.12.284.redhat-00002
     [java]     -   artifactId: jmespath-java
     [java]         groupId: com.amazonaws
     [java]         version: 1.12.284.redhat-00002
     [java]     -   artifactId: hppc
     [java]         groupId: com.carrotsearch
     [java]         version: 0.8.1.redhat-00001
     [java]     -   artifactId: classmate</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-1.liferay.com/job/test-portal-source-format/4487/">test-portal-source-format#4487</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-1.liferay.com/userContent/jobs/test-portal-source-format/builds/4487/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/marcoscv-work/liferay-portal/pull/370">marcoscv-work#370</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/226438307">[master] ci:test:sf - marcoscv-work &gt; marcoscv-work - PR#370 - 2025-05-27[05:33:19]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>226438307</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-1.liferay.com/job/publish-testray-report/29000/">publish-testray-report#29000</a><br/></div></div>

dsitu committed:
LPD-56336 Refactor CPCommerceShipments#CanCheckShipmentTrackingURLAfterShippingMethodSelection from poshi to playwright

dsitu committed:
LPD-36336 Remove poshi test

marcoscv-work commented:
ci:test:sf

liferay-continuous-integration commented:
Test suite: ci:test:sf has been triggered.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-3.liferay.com/job/test-portal-source-format/8730/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-3.liferay.com/userContent/jobs/test-portal-source-format/builds/8730/jenkins-report.html">here</a>.</b></p>

marcogalluzzi committed:
LPD-56159 Fix type error in test

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 35 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a0d9ed61a9e12f60eb54a55a75203decf0b30628">a0d9ed61a9e12f60eb54a55a75203decf0b30628</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/marcoscv-work/liferay-portal/tree/cms-theme-rebased-4">cms-theme-rebased-4</a><br/>Branch GIT ID: <a href="https://github.com/marcoscv-work/liferay-portal/commit/6deacdea84e84b31129c80c1560a049518f7313f">6deacdea84e84b31129c80c1560a049518f7313f</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-3.liferay.com/job/test-portal-source-format/8730/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-3.liferay.com/userContent/jobs/test-portal-source-format/builds/8730/jenkins-report.html">here</a>.</h5><pre>format-source-files:
     [java] Loading file:/opt/dev/projects/github/liferay-portal/portal-impl/classes/system.properties
     [java] java.lang.Exception: Found 1 formatting issues:
     [java] 1: ./workspaces/liferay-testray-workspace/modules/testray-rest-impl/build.gradle expected:&lt;...a.ws.rs", version: "[3.1.0.LIFERAY-PATCHED-1]"
     [java]   compileOnly group...&gt; but was:&lt;...a.ws.rs", version: "[2.1.6.LIFERAY-PATCHED-2]"
     [java]   compileOnly group...&gt;
     [java] 
     [java]   at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:474)
     [java]   at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:302)
[stopwatch] [run.batch.test.action: 33:11.233 sec]
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:421: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:623: Java returned: 1
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null803014660.properties

merge-test-results:
    [mkdir] Created dir: /opt/dev/projects/github/liferay-portal/test-results
[junitreport] Processing /opt/dev/projects/github/liferay-portal/test-results/TESTS-TestSuites.xml to /tmp/null1637973293
[junitreport] Loading stylesheet ile:/opt/java/ant/lib/ant-junit.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
[junitreport] Transform time: 296ms
[junitreport] Deleting: /tmp/null1637973293
     [echo] A report with all the test results can be found at test-results/html/index.html.
    [mkdir] Created dir: /opt/dev/projects/github/liferay-jenkins-ee/test-results
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-jenkins-ee/test-results
     [echo] run.batch.test.tear.down.start.timestamp: 05-28-2025 03:08:14:760 PDT
[stopwatch] [run.batch.test.tear.down: 0.000 sec]
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null803014660.properties

print-gc-logs:
     [echo] 
     [echo] ##
     [echo] ## /tmp/ant-gc.log
     [echo] ##
     [echo] 
[beanshell] Java HotSpot(TM) 64-Bit Server VM (25.381-b09) for linux-amd64 JRE (1.8.0_381-b09), built on Jun 14 2023 13:47:37 by "java_re" with gcc 7.3.0
[beanshell] Memory: 4k page, physical 32902044k(8603508k free), swap 1048572k(0k free)</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-3.liferay.com/job/test-portal-source-format/8730/">test-portal-source-format#8730</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-3.liferay.com/userContent/jobs/test-portal-source-format/builds/8730/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/marcoscv-work/liferay-portal/pull/370">marcoscv-work#370</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/227289576">[master] ci:test:sf - marcoscv-work &gt; marcoscv-work - PR#370 - 2025-05-28[02:33:00]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>227289576</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-3.liferay.com/job/publish-testray-report/5108/">publish-testray-report#5108</a><br/></div></div>

smottal committed:
LPD-55587 Service layer

smottal committed:
LPD-55587 Integration test

smottal committed:
LPD-55587 Update usage

smottal committed:
LPD-55587 Autogenerated Code (buildService)

marcoscv-work commented:
ci:test:sf

liferay-continuous-integration commented:
Test suite: ci:test:sf has been triggered.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-25.liferay.com/job/test-portal-source-format/3481/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-source-format/builds/3481/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 27 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/fb1c542ccd46c3136eb7c33be1a0c7b9b416db9b">fb1c542ccd46c3136eb7c33be1a0c7b9b416db9b</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/marcoscv-work/liferay-portal/tree/cms-theme-rebased-4">cms-theme-rebased-4</a><br/>Branch GIT ID: <a href="https://github.com/marcoscv-work/liferay-portal/commit/c7bb61438831bb21df5caba3496c62d76d6e1895">c7bb61438831bb21df5caba3496c62d76d6e1895</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-25.liferay.com/job/test-portal-source-format/3481/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-source-format/builds/3481/jenkins-report.html">here</a>.</h5><pre>format-source-files:
     [java] Loading file:/opt/dev/projects/github/liferay-portal/portal-impl/classes/system.properties
     [java] java.lang.Exception: Found 5 formatting issues:
     [java] 1: ./bun dles/jboss-eap-8.0.0/.installation/manifest_version.yaml expected:&lt;-[ groupId: org.jboss.eap.channels
     [java]     artifactId: eap-8.0
     [java]     description:
     [java]         "Manifest for EAP 8.0"
     [java]     version: 1.0.1.GA-redhat-00003
     [java] maven:
     [java] schemaVersion: 1.0.0]&gt; but was:&lt;-[--
     [java] schemaVersion: "1.0.0"
     [java] maven:
     [java] - groupId: "org.jboss.eap.channels"
     [java]   artifactId: "eap-8.0"
     [java]   version: "1.0.1.GA-redhat-00003"
     [java]   description: "Manifest for EAP 8.0"
     [java] ]&gt;
     [java] 2: ./bun dles/jboss-eap-8.0.0/.installation/installer-channels.yaml expected:&lt;[manifest:
     [java]     maven:
     [java]         artifactId: eap-8.0
     [java]         groupId: org.jboss.eap.channels
     [java] name: eap-8.0
     [java] noStreamStrategy: none
     [java] repositories:
     [java]     -   id: mrrc
     [java]         url: https://maven.repository.redhat.com/ga/
     [java] schemaVersion: 2.0.0]&gt; but was:&lt;[---
     [java] schemaVersion: "2.0.0"
     [java] name: "eap-8.0"
     [java] repositories:
     [java]   - id: "mrrc"
     [java]     url: "https://maven.repository.redhat.com/ga/"
     [java] manifest:
     [java]   maven:
     [java]     groupId: "org.jboss.eap.channels"
     [java]     artifactId: "eap-8.0"
     [java] noStreamStrategy: "none"
     [java] ]&gt;
     [java] 3: Truncated message :
     [java] ./bun dles/jboss-eap-8.0.0/.installation/manifest.yaml expected:&lt;[schemaVersion: 1.0.0
     [java] streams:
     [java]     -   artifactId: aws-java-sdk-core
     [java]         groupId: com.amazonaws
     [java]         version: 1.12.284.redhat-00002
     [java]     -   artifactId: aws-java-sdk-kms
     [java]         groupId: com.amazonaws
     [java]         version: 1.12.284.redhat-00002
     [java]     -   artifactId: aws-java-sdk-s3
     [java]         groupId: com.amazonaws
     [java]         version: 1.12.284.redhat-00002
     [java]     -   artifactId: jmespath-java
     [java]         groupId: com.amazonaws
     [java]         version: 1.12.284.redhat-00002
     [java]     -   artifactId: hppc
     [java]         groupId: com.carrotsearch
     [java]         version: 0.8.1.redhat-00001
     [java]     -   artifactId: classmate</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-25.liferay.com/job/test-portal-source-format/3481/">test-portal-source-format#3481</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-source-format/builds/3481/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/marcoscv-work/liferay-portal/pull/370">marcoscv-work#370</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/228454383">[master] ci:test:sf - marcoscv-work &gt; marcoscv-work - PR#370 - 2025-05-29[03:16:09]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>228454383</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-25.liferay.com/job/publish-testray-report/14373/">publish-testray-report#14373</a><br/></div></div>

marcoscv-work commented:
ci:test:sf

liferay-continuous-integration commented:
Test suite: ci:test:sf has been triggered.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-34.liferay.com/job/test-portal-source-format/3955/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-source-format/builds/3955/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 32 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/fa035483eaffb146260bdb93571d1f2a39b7c9bb">fa035483eaffb146260bdb93571d1f2a39b7c9bb</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/marcoscv-work/liferay-portal/tree/cms-theme-rebased-4">cms-theme-rebased-4</a><br/>Branch GIT ID: <a href="https://github.com/marcoscv-work/liferay-portal/commit/20133342149ba4d1667f6098c83f2d37f46f6428">20133342149ba4d1667f6098c83f2d37f46f6428</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-34.liferay.com/job/test-portal-source-format/3955/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-source-format/builds/3955/jenkins-report.html">here</a>.</h5><pre>     [exec] &gt; Task :packageRunCheckFormat
     [exec] yarn run v1.22.22
     [exec] \$ node-scripts check:ci
     [exec] 
     [exec] ⚙️ Running preflight checks...
     [exec] 
     [exec] ⚙️ Checking outdated tsconfig.json files ...
     [exec] 
     [exec] ⚙️ Running TypeScript checks on modified files...
     [exec] ℹ️ A total of 12 CPUs were detected: launching tsc using 12 workers
     [exec] ✅ Checked apps/site/site-cms-site-initializer
     [exec] 
     [exec] ⚙️ Running format checks on modified files...
     [exec]    /opt/dev/projects/github/liferay-portal/modules/apps/frontend-theme/frontend-theme-cms/src/css/_clay_variables.scss
     [exec]      1:1  error  File has format errors.  (format check)
     [exec]    
     [exec]    ✖ 1 problem (1 error, 0 warnings)
     [exec]    
     [exec]    /opt/dev/projects/github/liferay-portal/modules/apps/frontend-theme/frontend-theme-cms/src/css/components/_sticker.scss
     [exec]      9:25  error  Unexpected quotes (function-url-quotes)  function-url-quotes
     [exec]    
     [exec]    ✖ 1 problem (1 error, 0 warnings)
     [exec]    
     [exec]    
     [exec] ❌ CI checks failed.
     [exec] error Command failed with exit code 1.
     [exec] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
     [exec] 
     [exec] &gt; Task :packageRunCheckFormat FAILED
     [exec] Gradle build finished at 2025-05-29 17:58:02.309.
     [exec] 
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':packageRunCheckFormat'.
     [exec] &gt; Process 'command '/opt/dev/projects/github/liferay-portal/build/node/bin/node'' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] &gt; Run with --info or --debug option to get more log output.
     [exec] &gt; Run with --scan to get full insights.
     [exec] &gt; Get more help at https://help.gradle.org.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':packageRunCheckFormat'.
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda\$executeIfValid\$1(ExecuteActionsTaskExecuter.java:148)
     [exec]   at org.gradle.internal.Try\$Failure.ifSuccessfulOrElse(Try.java:282)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:146)</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-34.liferay.com/job/test-portal-source-format/3955/">test-portal-source-format#3955</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-source-format/builds/3955/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/marcoscv-work/liferay-portal/pull/370">marcoscv-work#370</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/228785966">[master] ci:test:sf - marcoscv-work &gt; marcoscv-work - PR#370 - 2025-05-29[10:26:15]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>228785966</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-34.liferay.com/job/publish-testray-report/26464/">publish-testray-report#26464</a><br/></div></div>

marcoscv-work committed:
LPD-43525 - Add CMS Theme

marcoscv-work committed:
LPD-51911 - Add Sticker Dropdown for CMS

marcoscv-work committed:
LPD-55626 - Improve CMS Control Menu Styles and Apply CMS Theme

JavierMoral committed:
LPD-56394 Integration test to reproduce the issue

JavierMoral committed:
LPD-56394 Take into account ctCollectionId

brbalazs committed:
LPD-56738 Fix flaky tests

lfbesada committed:
LPD-56394 Extract field

lfbesada committed:
LPD-56394 Extends from BaseCTUpgradeProcessTestCase to assert it works OK on publications

smottal committed:
LPD-55587 Rewrite comments

ling-alan-huang committed:
LPD-44842 Rename

ling-alan-huang committed:
LPD-44842 Rename

ling-alan-huang committed:
LPD-44842 Rename

ling-alan-huang committed:
LPD-44842 Rename

ling-alan-huang committed:
LPD-44842 Rename

ling-alan-huang committed:
LPD-44842 Rename

JavierMoral committed:
LPD-56394 Rename class so it matches its base class

marcogalluzzi committed:
LPD-56159 Restoring creation option Multiple Upload

marcoscv-work committed:
LPD-43525 - Copyright Date Update

marcoscv-work committed:
LPD-43525 - Add lfrbuild-releng-ignore

ling-alan-huang committed:
LPD-44271 SF

ling-alan-huang committed:
LPD-44271 SF

ling-alan-huang committed:
LPD-44271 SF

ling-alan-huang committed:
LPD-44271 SF

ling-alan-huang committed:
LPD-44271 SF

ling-alan-huang committed:
LPD-44271 SF

ling-alan-huang committed:
LPD-44271 SF

ling-alan-huang committed:
LPD-44271 SF

ling-alan-huang committed:
LPD-44271 SF

ling-alan-huang committed:
LPD-44271 SF

ling-alan-huang committed:
LPD-44271 Auto SF

ling-alan-huang committed:
LPD-44271 SF

ling-alan-huang committed:
LPD-44271 SF

boton committed:
LPD-56916 Add option to skip default navigation after enabling the FF

ling-alan-huang committed:
LPD-44842 Rename

ling-alan-huang committed:
LPD-44842 Rename

ling-alan-huang committed:
LPD-44842 Rename

marianoalvarosaiz committed:
LPD-6627 Fix to adapt to changed message.

dsanz committed:
LPD-55693 Sort and reword

brianchandotcom committed:
Baseline

ling-alan-huang committed:
LPD-55602 Auto SF

brianchandotcom committed:
LPD-56394 SF

brianchandotcom committed:
LPD-56394 SF

brianchandotcom committed:
LPD-55587 Consistency

smottal committed:
LPD-57012 Remove poshi tests already covered by integrations

brianchandotcom committed:
LPD-56159 Sort

brianchandotcom committed:
LPD-56159 SF

marcoscv-work committed:
LPD-43525 - Use Relative Path

marcoscv-work committed:
LPD-55626 - Sort
